flink-staging/flink-hcatalog/pom.xml

```diff
@@ -34,17 +34,64 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<repositories>
+		<repository>
+			<id>cloudera</id>
+			<url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
+		</repository>
+	</repositories>
+
```

Comment:
I'm not a HCatalog expert. But I'm not sure that this 3rd-party repository is needed.